py4DSTEM/__init__.py

```diff
@@ -52,18 +52,74 @@
     BraggVectorMap,
 )
 
+from py4DSTEM.process import classification
+
+# import (
+#     BraggVectorClassification,
+#     ConvergenceWarning,
+#     DataCube,
+#     FastICA,
+#     Featurization,
+#     GaussianMixture,
+#     MinMaxScaler,
+#     NMF,
+#     PCA,
+#     PointListArray,
+#     RobustScaler,
+#     permutations,
+# )
+
+# diffraction
+from py4DSTEM.process.diffraction import Crystal, Orientation
+
+# py4DSTEM.process.diffraction
+# ['Axes', 'Crystal', 'Crystal_Phase', 'Figure', 'Fraction', 'ListedColormap', 'Orientation', 'PointList', 'PointListArray', 'single_atom_scatter']
+
+
+# ptycho
+from py4DSTEM.process import phase
+
+# import (
+#     DPCReconstruction,
+#     MixedstatePtychographicReconstruction,
+#     MultislicePtychographicReconstruction,
+#     OptimizationParameter,
+#     OverlapMagneticTomographicReconstruction,
+#     OverlapTomographicReconstruction,
+#     ParallaxReconstruction,
+#     PtychographyOptimizer,
+#     SimultaneousPtychographicReconstruction,
+#     SingleslicePtychographicReconstruction,
+# )
+
+# polar
+from py4DSTEM.process.polar import PolarDatacube
+
+# from py4DSTEM.process import (
+#     PolarDatacube,
+# )
+
 # strain
 from py4DSTEM.process.strain.strain import StrainMap
 
-# TODO - crystal
-# TODO - ptycho
-# TODO - others
-
-# TODO - where
-from py4DSTEM.process import (
-    PolarDatacube,
-)
-
+# TODO WPF
+from py4DSTEM.process import wholepatternfit
```

Comment:
@sezelt the wholepatternfit namespace is just two `import *` statements - this would be great to tidy at some point :p